Bottom: e2c0014c734beb600d34d764ea62c999ea687c38
Top:    a53de8f238f7910f979bb53f53f025f799110f93
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-02-04 17:07:14 -0800

Refresh of rstream-cnt

---

diff --git a/examples/common.c b/examples/common.c
index 294dba1..28c0fb6 100644
--- a/examples/common.c
+++ b/examples/common.c
@@ -117,11 +117,11 @@ void cnt_str(char *str, size_t ssize, long long cnt)
 
 int size_to_count(int size)
 {
-	if (size >= 1000000)
+	if (size >= (1 << 20))
 		return 100;
-	else if (size >= 100000)
+	else if (size >= (1 << 16))
 		return 1000;
-	else if (size >= 10000)
+	else if (size >= (1 << 11))
 		return 10000;
 	else
 		return 100000;
